docs/src/gs/create-workload-cluster.md

```diff
@@ -6,41 +6,47 @@ Choose one of the available templates for to create your workload clusters from
+| `OCI_CONTROL_PLANE_PV_TRANSIT_ENCRYPTION` | true                | [In-transit encryption](https://docs.oracle.<br/>com/en-us/iaas/Content/File/Tasks/intransitencryption.htm) provides a way to secure your data between instances and mounted file systems using TLS 1.2 (Transport Layer Security) encryption. Only [some bare metal instances](https://docs.oracle.com/en-us/iaas/releasenotes/changes/60d602f5-abb3-4639-aa19-292a5744a808/) support in-transit encryption |
+| `OCI_WORKER_PV_TRANSIT_ENCRYPTION`        | true                | [In-transit encryption](https://docs.oracle.com/en-us/iaas/Content/File/Tasks/intransitencryption.htm) provides a way to secure your data between instances and mounted file systems using TLS 1.2 (Transport Layer Security) encryption. Only [some bare metal instances](https://docs.oracle.com/en-us/iaas/releasenotes/changes/60d602f5-abb3-4639-aa19-292a5744a808/) support in-transit encryption      |
+
+*NOTE:* Using the predefined templates the machine's memory size is automatically allocated based on the chosen shape 
+and OCPU count.
 
 The following Cluster API parameters are also available:
 
-| Parameter                      | Default Value          | Description |
-| ----------------------------   | ---------------------- | ----------- |
-| `CLUSTER_NAME`                 |                        | The name of the workload cluster to create |
-| `CONTROL_PLANE_MACHINE_COUNT`  |       1                | The number of control plane machines for the workload cluster.|
-| `KUBERNETES_VERSION`           |                        | The Kubernetes version to use for the workload cluster. If unspecified, the value from OS environment variables or the .cluster-api/clusterctl.yaml config file will be used. |
-| `NAMESPACE`                    |                        | The namespace to use for the workload cluster. If unspecified, the current namespace will be used |
-| `POD_CIDR`                     |       1                | The CIDR range for the Kubernetes POD network. |
-| `SERVICE_CIDR`                 |                        | The CIDR for the Kubernetes services network.  |
-| `SERVICE_DOMAIN`               |                        |  |
-| `WORKER_MACHINE_COUNT`         |                        | The number of worker machines for the workload cluster. |
+| Parameter                      | Default Value  | Description |
+| ----------------------------   |----------------| ----------- |
+| `CLUSTER_NAME`                 |                | The name of the workload cluster to create |
+| `CONTROL_PLANE_MACHINE_COUNT`  | 1              | The number of control plane machines for the workload cluster.|
+| `KUBERNETES_VERSION`           |                | The Kubernetes version to use for the workload cluster. If unspecified, the value from OS environment variables or the .cluster-api/clusterctl.yaml config file will be used. |
```

Comment:
```suggestion
| `KUBERNETES_VERSION`           |                | The Kubernetes version installed on the workload cluster nodes. If this environement variable is not configured, the version must be specifed in the `.cluster-api/clusterctl.yaml` file |
```

The above suggestion is not correct if the assumption I made based on the format of the variable names that the table is defining the use of the each parameter when used as an environment variable is not true

Reply:
@shyamradhakrishnan might be a better person to answer this. Which is the same as `main` https://github.com/oracle/cluster-api-provider-oci/blob/main/docs/src/gs/create-workload-cluster.md?plain=1#L27